core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -89,71 +83,46 @@ public int run(final String[] args, final Properties config) {
         consumerConfig.clear();
         consumerConfig.putAll(config);
 
-        int exitCode = EXIT_CODE_SUCCESS;
-
-        AdminClient adminClient = null;
-        ZkUtils zkUtils = null;
-        try {
-            parseArguments(args);
-            dryRun = options.has(dryRunOption);
+        parseArguments(args);
 
-            adminClient = AdminClient.createSimplePlaintext(options.valueOf(bootstrapServerOption));
-            final String groupId = options.valueOf(applicationIdOption);
+        consumerConfig.setProperty(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, options.valueOf(bootstrapServerOption));
 
+        int exitCode = EXIT_CODE_SUCCESS;
 
-            zkUtils = ZkUtils.apply(options.valueOf(zookeeperOption),
-                30000,
-                30000,
-                JaasUtils.isZkSecurityEnabled());
+        try (AdminClient adminClient = KafkaAdminClient.create(consumerConfig)){
+            dryRun = options.has(dryRunOption);
 
             allTopics.clear();
-            allTopics.addAll(scala.collection.JavaConversions.seqAsJavaList(zkUtils.getAllTopics()));
-
-
-            if (!adminClient.describeConsumerGroup(groupId, 0).consumers().get().isEmpty()) {
-                throw new IllegalStateException("Consumer group '" + groupId + "' is still active. " +
-                            "Make sure to stop all running application instances before running the reset tool.");
-            }
+            allTopics.addAll(adminClient.listTopics().names().get());
 
             if (dryRun) {
                 System.out.println("----Dry run displays the actions which will be performed when running Streams Reset Tool----");
             }
+
             maybeResetInputAndSeekToEndIntermediateTopicOffsets();
-            maybeDeleteInternalTopics(zkUtils);
+            maybeDeleteInternalTopics(adminClient);
 
         } catch (final Throwable e) {
             exitCode = EXIT_CODE_ERROR;
             System.err.println("ERROR: " + e.getMessage());
-        } finally {
-            if (adminClient != null) {
-                adminClient.close();
-            }
-            if (zkUtils != null) {
-                zkUtils.close();
-            }
         }
 
         return exitCode;
     }
 
-    private void parseArguments(final String[] args) throws IOException {
+    private void parseArguments(final String[] args) {
 
         final OptionParser optionParser = new OptionParser(false);
         applicationIdOption = optionParser.accepts("application-id", "The Kafka Streams application ID (application.id).")
             .withRequiredArg()
             .ofType(String.class)
             .describedAs("id")
             .required();
-        bootstrapServerOption = optionParser.accepts("bootstrap-servers", "Comma-separated list of broker urls with format: HOST1:PORT1,HOST2:PORT2")
+        bootstrapServerOption = optionParser.accepts("bootstrap-server", "Comma-separated list of broker urls with format: HOST1:PORT1,HOST2:PORT2")
```

Comment:
Why removing the trailing 's' ?

Reply:
Forgot to mention it on the KIP, was thinking about aligning options. In Consumer Group command, and other, the option is `bootstrap-server`. Should I rollback this change?

Reply:
If it is not in KIP, better roll back.

Reply:
yeap, pushing changes.